datafusion-cli/Cargo.toml

```diff
@@ -39,3 +39,6 @@ object_store = { version = "0.5.0", features = ["aws", "gcp"] }
 rustyline = "10.0"
 tokio = { version = "1.0", features = ["macros", "rt", "rt-multi-thread", "sync", "parking_lot"] }
 url = "2.2"
+
+[dev-dependencies]
+serial_test = "*"
```

Comment:
Rather than a new dependency, could we just merge them into a single test?

Reply:
In this case, yes. As per your recommendation, I merged two test into one.